content/acra/configuring-maintaining/general-configuration/acra-authmanager.md

```diff
@@ -110,20 +111,23 @@ read an encrypted file with authentication data.
 
 * `--vault_tls_client_ca=<filename>`
 
-  Path to AcraServer TLS certificate's CA certificate for Vault certificate validation (AcraServer works as "client" when communicating with Vault).
+  Path to acra-authmanager TLS certificate's CA certificate for Vault certificate validation (AcraServer works as "client" when communicating with Vault).
   Empty by default.
+  If not specified, acra-authmanager uses value from `--tls_ca` flag.
 
 
 * `--vault_tls_client_cert=<filename>`
 
-  Path to AcraServer TLS certificate presented to Vault (AcraServer works as "client" when communicating with Vault).
+  Path to acra-authmanager TLS certificate presented to Vault (AcraServer works as "client" when communicating with Vault).
   Empty by default.
+  If not specified, acra-authmanager uses value from `--tls_cert` flag.
 
 
 * `--vault_tls_client_key=<filename>`
 
-  Path to AcraServer TLS certificate's private key of the TLS certificate presented to Vault (AcraServer works as "client" when communicating with Vault).
+  Path to acra-authmanager TLS certificate's private key of the TLS certificate presented to Vault (acra-authmanager works as "client" when communicating with Vault).
```

Comment:
Let's define a correct version.

Sometimes you use:

```
Path to acra-authmanager TLS certificate's private key of the TLS certificate presented to Vault (acra-authmanager works as "client" when communicating with Vault).
```

Other times you use:

```
Path to acra-authmanager TLS certificate's CA certificate for Vault certificate validation (AcraServer works as "client" when communicating with Vault).
```

I guess, first one is the correct and should be corrected.

Reply:
it is two different options and descriptions. one for private key related to `*_tls_*_key` flags, and second one to `*_tls_*_ca` certificate used to validate peer's certificate. And there is one more `*_tls_*_cert` flag related to the certificate paired with private key and presented to peer.
Or I misunderstood and you meant something else?